eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/AbstractBag.java

```diff
@@ -541,7 +541,7 @@ public long sumOfLong(LongFunction<? super T> function)
         long[] sum = {0L};
         this.forEachWithOccurrences((each, occurrences) -> {
             long longValue = function.longValueOf(each);
-            sum[0] += longValue * (long) occurrences;
+            sum[0] += longValue * occurrences;
```

Comment:
This change seems to be unrelated to this PR. Can you please revert it? Or it was due to some checkstyle violation?

Reply:
This was an unwanted change and it has now been reverted. I was trying out a checkstyle plugin in Eclipse, sorry :)